tools/backups2datalad/__main__.py

```diff
@@ -180,5 +181,70 @@ def release(
         dataset.push(to="github", jobs=datasetter.config.jobs)
 
 
+@main.command()
+@click.option("--backup-remote", help="Name of the rclone remote to push to")
+@click.option(
+    "-e",
+    "--exclude",
+    help="Skip dandisets matching the given regex",
+    metavar="REGEX",
+    type=re.compile,
+)
+@click.argument("backup_remote")
+@click.argument("dandisets", nargs=-1)
+@click.pass_obj
+def populate(
```

Comment:
let's adjust cron job script right away so it performs both actions to retain "parity" with current setup.  Later we will separate out the backup action to be ran independently (after figuring out how we should lock or whatever to avoid multiple processes operating on the same dandiset)

Reply:
Done.